src/tests/functional/loading_tests.ts

```diff
@@ -63,6 +63,20 @@ export class LoadingTests extends TurboDriveTestCase {
     const contents = await this.querySelector(frameContents)
     this.assert.equal(await contents.getVisibleText(), "Frames: #frame")
   }
+  
+  async "test reloading a frame reloads the content"() {
+    await this.nextBeat
+
+    await this.clickSelector("#loading-eager summary")
+    await this.nextBeat
+
+    const frameContent = "#loading-eager turbo-frame#frame h2"
+	this.assert.ok(await this.hasSelector(frameContent))
+	// @ts-ignore
```

Comment:
Why do we want this?

Reply:
because otherwise typescript complains that `reload` is not an Element method. 
![image](https://user-images.githubusercontent.com/1319150/111064863-c3462300-84b6-11eb-935e-af8b4f68c615.png)
shall I cast it instead?